ui/app/routes/jobs/run/templates/manage.js

```diff
@@ -0,0 +1,31 @@
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+
+export default class RunTemplatesRoute extends Route {
+  @service can;
+  @service router;
+  @service store;
+
+  beforeModel() {
+    const hasPermissions = this.can.can('write variable', null, {
+      namespace: '*',
+      path: '*',
+    });
+
+    if (!hasPermissions) {
+      this.router.transitionTo('jobs');
+    }
+  }
+
+  async model() {
+    const jobTemplateVariables = await this.store.query('variable', {
+      prefix: 'nomad/job-templates',
+      namespace: '*',
+    });
+    const recordsToQuery = jobTemplateVariables.map((template) =>
+      this.store.findRecord('variable', template.id)
+    );
```

Comment:
Seeing all variables [here](https://nomad-storybook-and-ee1v8mego-hashicorp.vercel.app/ui/jobs/run/templates/manage/) in the Mirage mocks: 
<img width="1104" alt="image" src="https://user-images.githubusercontent.com/713991/213086531-469c8eaf-4d2b-41d8-8dfc-a8e7fa8bfcee.png"> - is `prefix` the correct property, or is `path`?

Reply:
Note: also seeing more variables than expected on the non-management list page (templates index)

Reply:
In `mirage` mocks you will see all the variables here because we're using the `/vars` endpoint which does not have any logic for accounting for the `prefix`. Touching our `Mirage` setup would spill over into the `variables` view so I'm not going to touch it until after all the work is complete and we don't have more feature requests or updates on the feature.

Its annoying but we'll have to rely on a Nomad server here.

Reply:
Hey! Get back to being on vacation and recharge your batteries!!!